src/components/Calculator.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import './Calculator.css';
+
+function Calculator() {
+  return (
+    <div className="calculator">
+      <div className="display">0</div>
+      <div className="buttons">
+        <div className="row">
+          <button type="button" className="button">AC</button>
+          <button type="button" className="button">+/-</button>
+          <button type="button" className="button">%</button>
+          <button type="button" className="button-operator">÷</button>
+        </div>
+        <div className="row">
```

Comment:
You have done an excellent job 💪 implementing this component. However,🙆‍♀️ based on the project requirements, it is necessary to have at least one child🙋‍♀️ component in the Calculator.js component. Incorporating a child component will enhance the modularity and structure of your code, making it easier to maintain. By separating specific functionalities into smaller, reusable components, you can improve code organization and facilitate future updates and modifications. 🙏Please check below project requirements.

![image](https://github.com/StevenWafeek/math-magicians/assets/63445083/f94b4b9e-b727-43fd-8c11-f11eeccc2a39)